<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kompsitor</class>
 <widget class="QMainWindow" name="Kompsitor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>740</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kompsitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>675</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>675</height>
    </size>
   </property>
   <widget class="ClickingLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>675</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>675</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>675</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>1</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/backgrund/images/backgrund.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelPlay">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/buttons/images/playButton.png</pixmap>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelPause">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/buttons/images/pauseButton.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelStop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/buttons/images/stopButton.png</pixmap>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelNoteHalf">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/half.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelNoteQuater">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/quater.png</pixmap>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelNoteEight">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>55</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/eight.png</pixmap>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelNoteQuaterrest">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/quaterrest.png</pixmap>
    </property>
   </widget>
   <widget class="ClickingLabel" name="labelNoteEgithrest">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/eightrest.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelNote">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>285</y>
      <width>931</width>
      <height>143</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/notes/images/notblad.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVolume">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::sub-page:Horizontal { background-color: #9F2425; }
QSlider::add-page:Horizontal { background-color: #333333; }
QSlider::groove:Horizontal { background: transparent; height:4px; }
QSlider::handle:Horizontal {width:18px; border-radius:18px; background:#000000; margin: -5px 0px -5px 0px;}</string>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickingLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
